pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/BaseMigrator.java

```diff
@@ -93,17 +93,7 @@ private Map<String, Object> replaceLegacyAltPidKeysWithGuids(Map<String, Object>
     @Override
     public void export() {
         Map<String, Object> dataByRealm = getDataByRealm();
-        List<String> participantsInTheStudy = null;
-        participantsInTheStudy = elasticSearch.getAllParticipantsInIndex(index);
-        logger.info("Found {} participants in ES index {} for instance {}",
-                participantsInTheStudy.size(), index, realm);
-        for (String ddpParticipantId : participantsInTheStudy) {
-            if (!dataByRealm.containsKey(ddpParticipantId)) {
```

Comment:
Confirming that this is the main change we talked about last week.  If the migrator doesn't mention the participant, the data at the given key gets overwritten.  Removing this means that whatever is in ES will stay untouched if the migrator doesn't include the participant.

Reply:
Yes, this is the main change that keeps us from erasing in the OS1/OS2 case.
Not sure I follow the rest of your statement, but we now prevent overwriting for the entities mentioned in the PR descriptions.